client/src/Components/CommonComponents/AutoCompleteTags/index.jsx

```diff
@@ -18,17 +18,21 @@ export default class AutoCompleteTags extends Component {
   componentDidMount() {
     // git the data form the back for skills and offer_type and setSatet for the skills and offer type
     // const selected = options.filter(item => _.find(fetched, id => id === item.id))
-    const { data, type } = this.props;
-    this.setState(() => {
-      let options;
-      if (type === 'skill') {
-        options = skills;
-      } else if (type === 'offer_type') {
-        options = offerType;
+    const { type } = this.props;
+    this.setState(
+      () => {
+        let options;
+        if (type === 'skill') {
+          options = skills;
+        } else if (type === 'offer_type') {
```

Comment:
@Fatmasiam  Is it possible for `type` to not equal 'skill' or 'offer_type'?  If so, you might need a fallback value for options (since it would end up as `undefined` currently).

Reply:
we only have two types for AutoComplete
do we still need to do the fallback?

Reply:
If you know that it'll only be these two types, then I think no fallback is necessary